extensions/desktop/gnome/Makefile

```diff
@@ -17,14 +17,15 @@ clean:
 
 $(DEST_LAUNCHER):
 	@cat $(SRC_DIR)/init >> $(DEST_LAUNCHER)
-	# tail -n +2 to remove the shebang
+# tail -n +2 to remove the shebang
```

Comment:
lost the tab?

Reply:
the tab showed as a syntax error on emacs so I removed it, yes.

Reply:
Now it looks like a syntax error :P I'm not sure the comment is really necessary anyhow.

If these scripts are not intended to be used as-is, perhaps it makes sense to just drop the shebangs from the source?